infoschema/tables.go

```diff
@@ -1356,13 +1356,21 @@ func dataForTables(ctx sessionctx.Context, schemas []*model.DBInfo) ([][]types.D
 				if rowCount != 0 {
 					avgRowLength = dataLength / rowCount
 				}
+				var tableType string
+				switch schema.Name.L {
+				case util.InformationSchemaName.L, util.PerformanceSchemaName.L,
```

Comment:
how about MySQL? also, treats these as VIEW?

Reply:
For MySQL, tables in `information_schema` are SYSTEM_VIEWs, but tables in `performance_schema` is marked as `BASE`.

Theere was a discussion on this before...and we treat this as a bug of MySQL.